### PR TITLE
Render playlist description based on condition

### PR DESCRIPTION
**Pull request by AmbrishRamachandiran**
Render playlist description based on condition instead showing empty DOM

image

### CODE REVIEW

File: .changeset/ninety-hats-kiss.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-playlist': patch
+---
+
+Render playlist description based on condition
```
Comment (kuangp) [inline .changeset/ninety-hats-kiss.md:5]:
```suggestion
Conditionally render playlist description only when it's non-empty
```

**Review by kuangp — CHANGES_REQUESTED** [review]
Looks good - just minor suggestion to clarify the changeset description

File: .changeset/ninety-hats-kiss.md (hunk 1 above)
Comment (AmbrishRamachandiran) [reply .changeset/ninety-hats-kiss.md]: > 

Done

**Review by kuangp — APPROVED** [review]
Thanks!

**Pull request merged**